File: nvxapp.client/src/nvx/ClientServer-Service/GestionePresenze/Az_SediAttivita/Models/az-sediattivita-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";

export class Az_SediAttivitaModel {
  public id!: number;
  public idAz_Sedi!: number;
  public idPar_Attivita!: number;
}

export class Az_SediAttivita_GetAll_InModel {}

export class Az_SediAttivita_GetAll_OutModel extends ModelResult {
  public az_SediAttivita: Az_SediAttivitaModel[] = [];
}

File: nvxapp.server.service/ClientServer-Service/GestionePresenze/Az_SediAttivitaService/Models/Az_SediAttivitaModel.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;

namespace nvxapp.server.service.ClientServer_Service.GestionePresenze.Az_SediAttivitaService.Models
{
    public class Az_SediAttivitaModel
    {
        public int Id { get; set; }
        public int IdAz_Sedi { get; set; }
        public int IdPar_Attivita { get; set; }
    }

    public class Az_SediAttivita_GetAll_InModel { }

    public class Az_SediAttivita_GetAll_OutModel : ModelResult
    {
        public List<Az_SediAttivitaModel> Az_SediAttivita { get; set; } = new List<Az_SediAttivitaModel>();
    }
}
